client/src/service/api.ts

```diff
@@ -0,0 +1,27 @@
+export const URL = "http://localhost:3000";
```

Comment:
This also needs to be configurable (using 3000 and/or 3001 are inconvenient for me). Doesn't need to change in this PR, but it would be nice if we could get this updated soon.